spec/my_order_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'my_order'
+
+describe MyOrder do
+
+  before(:each) do
+    allow(subject).to receive(:actual_total) { :calculated_total }
```

Comment:
I think you should stub doubles and not the actual test subject, otherwise the class code would not run and tested.